scribus/pageitem_latexframe.cpp

```diff
@@ -121,6 +121,8 @@ void PageItem_LatexFrame::deleteImageFile()
 void PageItem_LatexFrame::DrawObj_Item(ScPainter *p, QRectF e)
 {
 	layout();
+	const QFont &font = QApplication::font();
+	p->setFont(PrefsManager::instance()->appPrefs.fontPrefs.AvailFonts.findFont(font.family(), font.styleName()), font.pointSizeF());

```

Comment:
I’d rather move the setFont call just before drawText, even if it is duplicated, to avoid setting a font needlessly.